routes/account.js

```diff
@@ -343,6 +354,12 @@ exports.registerpost = function(req, res) {
                       res.render('login', { title: "Login / Sign up", user: req.user,
                           errormessages:req.flash('error'), infomessages:req.flash('info') });
                     } else {
+                        if (!UserPassword.isComplexEnough(req.body.password)) {
```

Comment:
This should be the place where a user selected a password during registration, isn't it?